build/commands/lib/util.js

```diff
@@ -471,7 +471,6 @@ const util = {
       'import("//brave/tools/redirect_cc/args.gni")': null,
       use_remoteexec: config.useRemoteExec,
       use_reclient: config.useRemoteExec,
-      use_siso: false,
```

Comment:
pls keep `use_siso` in js both here and in redirect_cc. This will depend on `.env` in the near future.

Reply:
Done.